src/styles/blocks/header.scss

```
@@ -0,0 +1,59 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background: hsl(0, 0%, 100%);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.nav {
+  display: flex;
+  position: flex-end;
+  text-transform: uppercase;
+  margin: 0;
+  
+  &__item {
+    text-decoration: none;
+    display: block;
+    position: relative;
+    line-height: 60px;
+    font-weight: 500;
+  }
+  
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+  
+  &__item:hover {
+    color: var(--hover-color);
+    cursor: pointer;
+  }
+  
+}
+
+.is-active {
+  color: var(--hover-color);
```

you have another variable in scss
```suggestion
  color: $hoverColor;
```